extensions/amp-autocomplete/0.1/test/test-amp-autocomplete-bindings.js

```diff
@@ -0,0 +1,288 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-autocomplete';
+import {AutocompleteBindingInline} from '../autocomplete-binding-inline';
+import {AutocompleteBindingSingle} from '../autocomplete-binding-single';
+import {toggleExperiment} from '../../../../src/experiments';
+
+describes.realWin(
+  'amp-autocomplete bindings',
+  {
+    amp: {
+      extensions: ['amp-autocomplete'],
+    },
+  },
+  env => {
+    let win, doc, input;
+
+    beforeEach(() => {
+      win = env.win;
+      doc = win.document;
+      input = doc.createElement('input');
+    });
+
+    function getBindingStub(attributes) {
```

Comment:
1. nit: `createElementWithAttributes`
2. nit: async/await
3. Why are we instantiating an implementation object? Shouldn't we mock it instead?

Reply:
Thanks for this advice, the change for 3 made 2 unnecessary. 👍